components/style/values/computed/outline.rs

```diff
@@ -0,0 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
`*/` should move to next line?

Reply:
I think maybe it's okay.
I just copied this License from other files.